Problem2.py

```diff
@@ -0,0 +1,86 @@
+from collections import deque
+class Solution:
+	def orangesRotting1(self, grid):
+		# Used an approach similar to the game of life problem
+		# But if its not possible to convert everything to stale don't know the stopping condition
```

Comment:
the loop ends when we have iterated through all the stale oranges we have because after that no new oranges can be affected.

Reply:
But in my code I'm looping through all the elements in the grid. So I'm not understanding this iteration over all the stale oranges